priv/companies/trio.exs

```diff
@@ -0,0 +1,17 @@
+  %{
+    name: "Trio",
+    website: "https://www.triohq.com/en",
+    github: "https://github.com/trioHQ",
+    # reference lib/companies/industries.ex for a list of recommended industries to use here
+    industry: "Financials",
+    location: %{
+      city: "Curitiba",
+      state: "Paraná",
+      country: "Brazil"
+    },
+    description: """
+    Trio is a financial institution specializing in instant payments in Brazil. All our products, from the banking core to the payment checkout, were developed using Elixir, Phoenix, and LiveView.
+    """,
+    last_changed_on: ~D[2024-11-15]
+  }
+
```

Comment:
```suggestion
```suggestion
%{
  name: "Trio",
  website: "https://www.triohq.com/en",
  github: "https://github.com/trioHQ",
  # reference lib/companies/industries.ex for a list of recommended industries to use here
  industry: "Financials",
  location: %{
    city: "Curitiba",
    state: "Paraná",
    country: "Brazil"
  },
  description: """
  Trio is a financial institution specializing in instant payments in Brazil. All our products, from the banking core to the payment checkout, were developed using Elixir, Phoenix, and LiveView.
  """,
  last_changed_on: ~D[2024-11-15]
}
```
```

Reply:
Thank you @jsegal205